garbage-collect/handler.go

```diff
@@ -12,11 +12,23 @@ import (
 	"time"
 
 	"github.com/alexellis/hmac"
+	faasSDK "github.com/openfaas/faas-cli/proxy"
 	"github.com/openfaas/openfaas-cloud/sdk"
 )
 
 const Source = "garbage-collect"
 
+//FaaSAuth Authentication type for OpenFaaS
+type FaaSAuth struct {
```

Comment:
This would be good in the sdk

Reply:
cc @martindekov

Reply:
yes I have added the code in SDK .. as this is WIP PR and I have not cleaned it .

Reply:
We might need to first merge that SDK code and release it and then use that.

Reply:
Is copy paste this in the other functions a viable option until we release the new SDK containing this ^ ?